src/brod_group_coordinator.erl

```diff
@@ -579,13 +588,15 @@ sync_group(#state{ groupId       = GroupId
       [format_assignments(TopicAssignments)]),
   start_offset_commit_timer(NewState).
 
--spec handle_ack(state(), brod:topic(), brod:partition(), brod:offset()) ->
-        {ok, state()}.
-handle_ack(#state{ acked_offsets = AckedOffsets
-                 } = State, Topic, Partition, Offset) ->
+-spec handle_ack(state(), brod:group_generation_id(), brod:topic(), brod:partition(), brod:offset()) ->
```

Comment:
Very long line. We have (unfortunately unwritten) 80 char rule, that will break our internal CI. I will create an Elvis check some day

Reply:
Elvis actually did check it, that's why the CI failed. :D

It may be worth though adding an elvis command to make, so it is easier to get this feedback locally!